app/src/main/java/com/yacgroup/yacguide/database/AppDatabase.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2019 Fabian Kantereit
+ * Copyright (C) 2019 Axel Paetzold
```

Comment:
Please add yourself for year 2023 since you've added `exportSchema = true`

Reply:
I just fixed the original author. I did just a small change. I can add me on addition to you.

Reply:
I would add you since you have changed the app's behavior on building. If you only rename stuff, I'd agree to not change the Copyright header.